tests/cli/test_utils.py

```diff
@@ -197,11 +197,19 @@ def test_get_prompt_analyzer():
         '''
     """This is a comment"""
     foo = "bar"
+    number = 1
+    settings = Settings()
+    a_list = [1, 2, 3]
     '''
     )
     analyzer = get_prompt_analyzer(sample_file_content)
     assert analyzer.comments == "This is a comment"
-    assert analyzer.variables == {"foo": "bar"}
+    assert analyzer.variables == {
```

Comment:
just to make sure, when running `template.render` do we get back
```
number = 1
settings = Settings()
a_list = [1, 2, 3]
 ```

Reply:
yes, the items in the dictionary thats passed in are all wrapped in an extra layer of string so you have to put that in this specific test, but when you render it with `template.render` it gets peeled away hence everything being a string and actual strings being double quoted